pkgs/test/lib/src/runner/browser/chrome.dart

```diff
@@ -122,17 +122,19 @@ class Chrome extends Browser {
   Future<Uri> _sourceUriProvider(String sourceUrl, String scriptId) async {
     var script = _idToUrl[scriptId];
     if (script == null) return null;
-    var uri = Uri.parse(script);
+    // The script is a URL path so it should contain forward slashes.
+    var pathSegments = script.split('/');
     var path = p.join(
-        p.joinAll(uri.pathSegments.sublist(1, uri.pathSegments.length - 1)),
-        sourceUrl);
+        p.joinAll(pathSegments.sublist(1, pathSegments.length - 1)), sourceUrl);
```

Comment:
this could probably be `p.joinAll(pathSegments.skip(1))` or `p.joinAll(uri.pathSegments.skip(1)`